srcpkgs/pipewire/patches/with-bluetooth.patch

```diff
@@ -0,0 +1,19 @@
+Activate bluetooth support only when the bluetooth plugin is installed.
+--- src/daemon/media-session.d/media-session.conf.orig	2021-04-06 17:05:05.935876278 +0200
++++ src/daemon/media-session.d/media-session.conf	2021-04-06 17:06:16.599878154 +0200
+@@ -94,12 +94,14 @@
+     with-alsa = [
+         with-audio
+     ]
++    with-bluetooth = [
++        bluez5
++    ]
```

Comment:
Can this be upstreamed? Sounds like a nice improvement overall...

Reply:
ok I can propose it

Reply:
hmm I'm not entirely convinced. There are other plugins that do not throw errors even if enabled... I have to ask why. Please don't merge yet

Reply:
Upstream decreased the log priority of the message, so it's not an error anymore, and we don't need to change the config. I have backported the patch for this, which is already upstream.